src/components/QuantitySelector.js

```diff
@@ -0,0 +1,58 @@
+import styled from "styled-components";
+import { FaPlus, FaMinus } from "react-icons/fa";
+
+const QuantityButton = styled.button`
+    background: ${(p) => p.theme.colors.background};
+    color: ${(p) => p.theme.colors.text};
+    border: none;
+`
+
+const handleDecrease = (value, setValue) => {
+    if (value === 0) return; 
+    setValue(value - 1)
+}
+
+const handleIncrease = (value, setValue) => {
+    setValue(value + 1)
```

Comment:
```suggestion
    setValue(value + 1);
```
maybe we should install eslint?